Allows the user to change the avatar

AceLewis opened the pull request:
Allows the user change the avatar from the default one given, it uses a invisible
1*1 pixel png as a placeholder so the broken image icon is not
displayed.

AceLewis committed:
Allows the user to change the avatar

Allows the user change the avatar from the default one given, it uses a
1*1 pixel png as a placeholder so the broken image icon is not
displayed.

HelloZeroNet commented:
Including non-existent files slow down your site load time, so please add only .png

AceLewis committed:
Noted, only png to avoid timeout

Removed jpeg and gifs to avoid the browser timeout. (gif could be added
in the same way as before as gifs can support opacity however I don't
think many people will want to use them)

AceLewis commented:
I now have it only use .png, I can't think of how to allow both .png's and .jpg's so it is simple for the end user to use both of them.

HelloZeroNet commented:
It could be possible by loading the content.json using fileGet API command and check if any of the files are in it
